schemas/com.mbta.ctd.subwayside.train-heartbeat.json

```diff
@@ -0,0 +1,394 @@
+{
+  "$id": "https://github.com/mbta/schemas/blob/main/schemas/com.mbta.ctd.subwayside.train-heartbeat.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "Train Heartbeat message received by the wayside server",
+  "type": "object",
+  "properties": {
+    "id": {
+      "$ref": "cloudevents.json#/properties/id",
+      "examples": ["200081266220230427135628421"]
+    },
+    "source": {
+      "$ref": "cloudevents.json#/properties/source",
+      "examples": ["hsctdprd.mbta.com/subwayside-12345"]
+    },
+    "specversion": {
+      "const": "1.0",
+      "$ref": "cloudevents.json#/properties/specversion"
+    },
+    "type": {
+      "const": "com.mbta.ctd.subwayside.train-heartbeat",
+      "$ref": "cloudevents.json#/properties/type"
+    },
+    "time": {
+      "$ref": "cloudevents.json#/properties/time"
+    },
+    "partitionkey": {
+      "type": "string",
+      "minLength": 1,
+      "description": "Key used to balance messages across multiple shards",
+      "examples": ["O200812662"]
+    },
+    "data": {
+      "description": "The event payload.",
+      "$ref": "#/$defs/datadef"
+    }
+  },
+  "required": [
+    "id",
+    "source",
+    "specversion",
+    "type",
+    "data",
+    "time",
+    "partitionkey"
+  ],
+  "$defs": {
+    "carNumber": {
+      "type": "string",
+      "description": "The 4 digit number of a car",
+      "pattern": "^\\d{4}$"
+    },
+    "timestamp": {
+      "description": "RFC3339 timestamp",
+      "type": "string",
+      "format": "date-time",
+      "pattern": "^\\d{4}-(0\\d|1[012])-([012]\\d|3[01])T([01]\\d|2[0-3]):([0-5]\\d):([0-5]\\d)([.]\\d{1,6})?(Z|[+-]\\d{1,2}:[0-5]\\d)$"
```

Comment:
question: is this validation pattern necessary? apparently JSON Schema already expects RFC 3339 datetimes: https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times

Reply:
`format` isn't required to be implemented by validators. From slightly above your link:
> By default, `format` is just an annotation and does not effect validation.